chips/lowrisc/src/hmac.rs

```diff
@@ -293,3 +290,28 @@ impl hil::digest::HMACSha512 for Hmac<'_> {
         Err(ErrorCode::NOSUPPORT)
     }
 }
+
+impl hil::digest::Sha256 for Hmac<'_> {
+    fn set_mode_sha256(&self) -> Result<(), ErrorCode> {
+        let regs = self.registers;
+
+        // Ensure the SHA is setup
+        regs.cfg.write(
+            CFG::HMAC_EN::CLEAR + CFG::SHA_EN::SET + CFG::ENDIAN_SWAP::SET + CFG::DIGEST_SWAP::SET,
+        );
+
+        Ok(())
+    }
+}
+
+impl hil::digest::Sha384 for Hmac<'_> {
```

Comment:
Do we eventually expected SHA384 and SHA512 support? If not, perhaps the virtualizer should not include them for now, since this is the only implementation.

Reply:
I do expect SHA384 and SHA512 in the future